spec/liquid_tags/stackblitz_tag_spec.rb

```diff
@@ -29,29 +26,76 @@ def generate_new_liquid(id)
       end.to raise_error(StandardError)
     end
 
-    it "accepts stackblitz id with a view parameter" do
-      expect do
-        generate_new_liquid(stackblitz_id_with_view)
-      end.not_to raise_error
```

Comment:
I removed these tests because the tests I added already proved that this class can parse without raising errors.